lib/Froxlor/Cron/Http/ApacheFcgi.php

```diff
@@ -56,7 +56,9 @@ protected function composePhpOptions(&$domain, $ssl_vhost = false)
 					// start block, cut off last pipe and close block
 					$filesmatch = '(' . str_replace(".", "\.", substr($filesmatch, 0, - 1)) . ')';
 					$php_options_text .= '  <FilesMatch \.' . $filesmatch . '$>' . "\n";
-					$php_options_text .= '  SetHandler proxy:unix:' . $domain['fpm_socket'] . '|fcgi://localhost' . "\n";
```

Comment:
not part of this feature. please put this in a separate PR

Reply:
It is neccessary for my server setup and I do nit have the time to add it manually after every merging. My git skills are not high enough how to handle that it is not included in the PR but always present in my repository